superset-frontend/src/features/datasets/styles.ts

```diff
@@ -25,22 +25,22 @@ export const StyledLayoutWrapper = styled.div`
   background-color: ${({ theme }) => theme.colors.grayscale.light5};
 `;
 
-const Column = styled.div`
+const Column = styled.div<{ width?: number }>`
```

Comment:
Given that `LeftColumn` and `RightColumn` are pretty much overriding the properties of `Column`, could you remove `Column`? I think it will be way more readable.

```
export const LeftColumn = styled.div<{ theme: SupersetTheme; width?: number }>`
  width: ${({ theme, width }) => width ?? theme.gridUnit * 80}px;
  height: auto;
  flex-direction: column;
`;

export const RightColumn = styled.div`
  width: auto;
  height: auto;
  display: flex;
  flex: 1 1 auto;
  flex-direction: column;
`;
```